### PR TITLE
[Community] community tentacles packages

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Commons/pull/185

### CODE REVIEW

File: octobot/community/authentication.py [hunk 1]
```diff
@@ -132,8 +153,16 @@ def _refresh_auth(self):
             CommunityAuthentication.REFRESH_TOKEN: self.refresh_token,
             CommunityAuthentication.GRANT_TYPE: CommunityAuthentication.REFRESH_TOKEN,
         }
-        resp = requests.post(self.authentication_url, params=params)
-        self._handle_auth_result(resp)
+        self._ensure_community_url()
+        try:
+            resp = requests.post(self.authentication_url, params=params)
+            self._handle_auth_result(resp)
+        except requests.ConnectionError as e:
```
Comment (Herklos) [inline octobot/community/authentication.py:160]: :+1:

File: octobot/community/community_analysis.py [hunk 2]
```diff
@@ -13,28 +13,57 @@
...
+                logger.exception(e, True, f"Error when getting community status : {e}")
+        await asyncio.gather(
+            get_stats(f"{bot_metrics_url}0/0/-1", "daily"),
+            get_stats(f"{bot_metrics_url}0/-1/0", "monthly"),
+            get_stats(f"{bot_metrics_url}0/0/0", "all")
+        )
+    # ugly workaround to prevent some of these current asyncio version warnings:
+    # "unclosed transport {'_pending_data': None, '_paused': False, '_sock': <socket.socket fd=-1,
+    # family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6>,"
+    await asyncio.sleep(0.01)
```
Comment (Herklos) [inline octobot/community/community_analysis.py:53]: Can `wait_asyncio_next_cycle` also do the job ?

Comment (GuillaumeDSM) [reply octobot/community/community_analysis.py:53]: unfortunatelly no, here it's an ussue regarding window's sockets closure details I didn't do much research about it yet but it's more complicated than this :/ (I tried it though, doesn't help)

File: octobot/community/community_analysis.py [hunk 3]
```diff
@@ -83,13 +128,35 @@ def _get_top_occurrences(item_by_occurrence, top_count):
...
+def _is_of_trader_type(item, trader_type):
+    if trader_type is TraderTypes.ALL:
+        return True
+    else:
+        session = item.get(community_fields.CommunityFields.CURRENT_SESSION.value, {})
+        if trader_type is TraderTypes.REAL \
+                and session.get(community_fields.CommunityFields.TRADER.value, False) is True:
+            return True
+        elif trader_type is TraderTypes.SIMULATED \
+                and session.get(community_fields.CommunityFields.TRADER.value, False) is False \
```
Comment (Herklos) [inline octobot/community/community_analysis.py:153]: Why not using 
```suggestion
                and not session.get(community_fields.CommunityFields.TRADER.value, False) \
```

**Review by Herklos — APPROVED** [review]

**Pull request merged**